src/text_processing_util_mds24/text_processing_util_mds24.py

```diff
@@ -5,6 +5,27 @@ def text_clean(docs: list[str]) -> list[str]:
 
 
 def frequency_vectorizer(docs: list[str]) -> list[dict[str:float]]:
+    """
+    Calculates the frequency of each word in a list of text documents.
+
+    Parameters
+    ----------
+    docs : list[str]
+        A list of text documents.
+
+    Returns
+    -------
+    list[dict[str: float]]
+        A list of dictionaries, where each dictionary contains the word and the word's frequency in a text document.
```

Comment:
Small fix to be more specific.

```suggestion
        A list of dictionaries, where each dictionary contains each word and its frequency in a text document.
```